upgrade_test.py

```diff
@@ -602,7 +602,11 @@ def test_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-stat
             stress_cmd_read_10m = self.params.get('stress_cmd_read_10m')
             read_10m_cs_thread_pool = self.run_stress_thread(stress_cmd=stress_cmd_read_10m)
 
-            InfoEvent(message='Sleeping for 60s to let cassandra-stress start before the upgrade...').publish()
+            InfoEvent(message='Starting s-b workload for 10m').publish()
+            stress_cmd_large_partitions_10m = self.params.get('stress_cmd_1')
```

Comment:
the name of this should be something like "large_partition_stress_during_upgrade"

Reply:
fixing